pom.xml

```diff
@@ -113,10 +113,10 @@
 
         <!-- Non-default maven plugin versions and configuration -->
         <version.jacoco.plugin>0.8.0</version.jacoco.plugin>
-        <version.org.jboss.galleon>3.0.1.Final</version.org.jboss.galleon>
+        <version.org.jboss.galleon>4.0.0.Alpha1</version.org.jboss.galleon>
         <version.org.wildfly.build-tools>1.2.10.Final</version.org.wildfly.build-tools>
         <version.org.wildfly.checkstyle-config>1.0.5.Final</version.org.wildfly.checkstyle-config>
-        <version.org.wildfly.galleon-plugins>3.0.1.Final</version.org.wildfly.galleon-plugins>
+        <version.org.wildfly.galleon-plugins>4.0.0.Alpha4</version.org.wildfly.galleon-plugins>
```

Comment:
Was it a coincidence that both org.jboss.galleon & org.wildfly.galleon-plugins were at 3.0.1.Final before?
Is there any relation between the 2 versions?

Reply:
@jmesnil , the difference in version is due to some issues found in galleon-plugins. The 2 galleon deps will be 4.0.Final